merqube_client_lib/templates/equity_baskets/decrement/create.py

```diff
@@ -58,26 +72,14 @@ def create(config: dict[str, Any], prod_run: bool = False) -> CreateReturn:
 
     index_info = cast(ClientDecrementConfig, index_info)
 
-    uin = index_info.underlying_index_name
-    if index_info.client_owned_underlying:
-        # check that the underlying index exists
-        logger.info(f"Checking that the underlying index {uin} exists")
-        try:
-            client.get_index_model(index_name=uin)
-        except APIError:
-            raise ValueError(f"{uin} is not a valid index or you are not permissioned for it")
-
-        underlying_secapi_metric = "price_return"
-    else:
-        logger.info(f"Checking that the underlying index {uin} is a permissioned MerQube provided TR")
-        tr = _get_trs(uin)[0]
-        underlying_secapi_metric = tr["metric"]
-
-    # set the metric to the TR metric
-    template.plot_metric = inner_spec.get("output_metric", (getattr(template, "plot_metric", "price_return")))
-
-    inner_spec["metric"] = underlying_secapi_metric  # the decrement index looks up the returns of the tr via this
-    inner_spec["underlying_ticker"] = uin
+    tr_name = _validate_dec_params(client, index_info)
+
+    inner_spec["holiday_spec"]["calendar_identifiers"] = [f"MQI:{tr_name}"]
+
+    inner_spec["start_date"] = index_info.start_date or index_info.base_date
```

Comment:
i have to override this because tis in the template, and if i only set base date they will diverge